tests/wpt/tests/html/semantics/embedded-content/media-elements/autoplay-disabled-by-csp.sub.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Why is this a .sub file? Am I missing something in the test that needs to be substituted at runtime?